pkg/builtins/bitwise.go

```diff
@@ -60,7 +60,7 @@ func (b *BitwiseBuiltinRunner) InitialStack() []memory.MaybeRelocatable {
 
 func (b *BitwiseBuiltinRunner) DeduceMemoryCell(address memory.Relocatable, mem *memory.Memory) (*memory.MaybeRelocatable, error) {
 	index := address.Offset % BITWISE_CELLS_PER_INSTANCE
-	if index < BIWISE_INPUT_CELLS_PER_INSTANCE {
+	if index < BITWISE_INPUT_CELLS_PER_INSTANCE {
```

Comment:
it seems like we still need to change the constant name in its definition in line 14